src/version/check.go

```diff
@@ -28,16 +33,32 @@ func NewChecker(parentCtx context.Context, logger *zap.SugaredLogger) *checker {
 func (c *checker) CheckIfLatest(currentVersion string) error {
 	defer c.cancel()
 
+	// Dev build
 	if currentVersion == DevVersionValue {
 		return fmt.Errorf(`skipped, found dev build`)
 	}
 
+	// Disabled by ENV
+	if value, _ := os.LookupEnv(EnvVersionCheck); strings.ToLower(value) == "false" {
+		return fmt.Errorf(fmt.Sprintf(`skipped, disabled by ENV "%s"`, EnvVersionCheck))
+	}
+
 	latestVersion, err := c.getLatestVersion()
 	if err != nil {
 		return err
 	}
 
-	if currentVersion != latestVersion {
+	current, err := semver.NewVersion(currentVersion)
+	if err != nil {
+		return err
+	}
+
+	latest, err := semver.NewVersion(latestVersion)
+	if err != nil {
+		return err
+	}
+
+	if latest.GreaterThan(current) {
```

Comment:
Teraz, ak by bola lokalna verzia z nejakeho dovodu vyssia, ako ta v Github Releases, tak je to OK.